### ISSUE TITLE
[Adaptive Cards - Home Page] - Accessible name defined for the 'Visit the hub' control doesn’t contain its complete visual name present on the page.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9302]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)
Tool: Developer Tool

**Repro Steps:**

1. Open the above URL on the Edge Dev browser.
2. Navigate to the 'Visit the hub' control present on the main section of the page using tab key.
3. Press F12 to open the developer tool and inspect the code.
4. Verify that, accessible name defined for the 'Visit the hub' control contain its complete visual name present on the page or not.

**Actual Result:**
Accessible name defined for the 'Visit the hub' control doesn’t contain its complete visual name present on the page. Currently aria-label="Click to navigate to the Adaptive Card Documentation Hub" is defined for the 'Visit the hub' control.

**Note:**
Same issue replicates for the 'Visit the hub' links present on the 'Schema Explorer', 'Samples' and 'Blog' page.

**Expected Result:**
Accessible name defined for the 'Visit the hub' control should contain its complete visual name present on the page. Aria-label= "Visit the hub" should be defined for the control and the remaining label part can be defined along with them.

**User Experience:**
User who relies on text-to-speech and speech input will be confused and disoriented while activating or accessing any controls on the page if accessible name of the controls does not contain their visual name.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG22/Understanding/label-in-name.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/dca3db43-844f-4c41-a4d2-0990cd2dc077)

https://github.com/user-attachments/assets/1571a874-ad55-4f3f-985a-82cc0ee9aff1

### Card JSON

```JSON
NA
```
